packages/gatsby-plugin-utils/src/is-feature-available.ts

```diff
@@ -0,0 +1,16 @@
+import type { AvailableFeatures } from "gatsby"
```

Comment:
Small change, feel free to ignore: Can we name this file the same as the exported function (`has-feature.ts`), or do you have other intentions for it?

Reply:
oh, yeah we should, to much iteration I guess 😅